zkdb/src/storage/merkle-tree-test.ts

```diff
@@ -0,0 +1,240 @@
+import {
+  Poseidon,
+  Field,
+  MerkleTree,
+  isReady,
+  shutdown,
+  Circuit,
+  Bool,
+} from 'snarkyjs';
+import * as IPFS from 'ipfs-core';
+import { NodesMap, BaseMerkleTree } from './trees/merkle/BaseMerkleTree.js';
+import MerkleTreeJSON from './trees/merkle/MerkleTreeJSON.js';
+import MerkleTreeIPDL from './trees/merkle/MerkleTreeIPDL.js';
+import crypto from 'crypto';
+
+const DEFAULT_HEIGHT = 20;
+
+(async function run() {
+  await isReady;
+
+  const storage = await IPFS.create();
```

Comment:
I think, this implementation is a bit out-date. Please consider to rebase and use the new implementation of `StorageEngineIPFS` instead of old `js-ipfs`.